lib/generators/rodauth/install_generator.rb

```diff
@@ -71,9 +74,13 @@ def create_mailer
         end
 
         def create_fixtures
-          test_unit_options = ::Rails.application.config.generators.options[:test_unit]
-          if test_unit_options[:fixture] && test_unit_options[:fixture_replacement].nil?
-            template "app/test/fixtures/accounts.yml"
+          test_unit_options = ::Rails.application.config.generators.options[:test_unit]          
+          if test_unit_options[:fixture] && test_unit_options[:fixture_replacement].nil?                                    
+            if ::Rails.application.config.generators.options[:rails][:test_framework] == :rspec            
+              copy_file "app/test/fixtures/accounts.yml", "spec/fixtures/accounts.yml"               
```

Comment:
Why are we using `copy_file` for RSpec branch, and `template` for minitest branch?

Reply:
Your right. I was using `copy_file` and forgot to switch it over. template would be more accurate. https://rdoc.info/github/wycats/thor/Thor%2FActions:template